content/_data/upgrades/2-440-1.adoc

```diff
@@ -0,0 +1,21 @@
+==== Upgrade to the latest version of Remoting
+
+The `-jnlpUrl ${JENKINS_URL}/computer/${AGENT_NAME}/jenkins-agent.jnlp` argument to the agent JAR has been deprecated; use `-url ${JENKINS_URL}` and `-name ${AGENT_NAME}` instead, potentially also passing in `-webSocket`, `-tunnel`, and/or work directory options as needed.
```

Comment:
For info, this line caused a few build and deploy errors. See https://github.com/jenkins-infra/jenkins.io/pull/7112

Reply:
Thanks.  I should have caught that during code review, but I don't remember seeing any failure message when I viewed that page.

Reply:
> Thanks. I should have caught that during code review, but I don't remember seeing any failure message when I viewed that page.

Yes, this is why I added a message. I'm not sure why did this not fails the PR checks. I wonder if there isn't a mechanism which would only include some snippet into jenkins.io after a certain date and time: that would explain since we had the LTS today

Reply:
That's very odd, thanks so much for resolving this @dduportal.
I didn't notice any failures in the checks so it definitely flew under my radar.